<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Stitching Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 上传图像按钮 -->
   <widget class="QPushButton" name="button_upload_image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_upload_image2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 2</string>
    </property>
   </widget>
   <!-- 检材图像显示 -->
   <widget class="QLabel" name="label_image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 样本图像显示 -->
   <widget class="QLabel" name="label_image2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 拼接图像显示 -->
   <widget class="QLabel" name="label_image_stitched">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>850</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Stitched Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 拼接线调整滑动条 -->
   <widget class="QSlider" name="slider_stitch_position">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>850</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <!-- 保存截图按钮 -->
   <widget class="QPushButton" name="button_save_screenshot">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>640</y>
      <width>60</width>
      <height>30</height>
      </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
